packages/camera/camera/example/lib/main.dart

```diff
@@ -130,9 +130,9 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
         ),
       );
     } else {
-      return AspectRatio(
-        aspectRatio: controller.value.aspectRatio,
-        child: CameraPreview(controller),
+      return Container(
+        // aspectRatio: controller.value.aspectRatio,
```

Comment:
Aspect ratio should be enabled:
```suggestion
        aspectRatio: controller.value.aspectRatio,
```